### PR TITLE
Mobile

### PR DESCRIPTION
**Pull request by Katrina-Dierking**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [x ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/users/MobileStyle.css [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+    div.ui.large.fluid.pointing.vertical.tabular.menu {
+        display:flex;
+        flex-direction:row;
+        margin-top:520px;
+
+        /* overflow:hidden; */
+        background: lightgrey;
+        position: fixed;
+        z-index: 1;
+        margin-left: -3.6%;
```
Comment (agyin3) [inline src/components/users/MobileStyle.css:20]: Rather than doing `margin-left: -3.6%` and `margin-top: 520px` try `left: 0` and `bottom: 0`

**Comment by agyin3** [comment]
Overall it's looking pretty solid! There does seem to be a weird bug where the regular menu is still showing up when viewing on an `iphone 6/7/8 plus`

**Comment by Jeffreyo3** [comment]
The styling is looking really good - especially the menu tabs on the user's profile dashboard!

Consider updating or adding a wider max-width mobile breakpoint for larger phones and 'plus' models. Mobile styling isn't working on the Pixel 3 or the iPhone 7plus.

Keep up the awesome design work!! 💃

File: src/components/users/MobileStyle.css (hunk 1 above)
Comment (Katrina-Dierking) [reply src/components/users/MobileStyle.css:20]: Thank you. I kept trying to do margin-left:0 and margin-bottom:0 and that wouldn't work. Thank you so much. You're a rock star!!!

**Comment by agyin3** [comment]
Profile page looks great on the mobile view

**Pull request merged**